snappy/configUpdate.py

```diff
@@ -11,7 +11,7 @@ def configUpdate(original, update):
             if not isinstance(update[key], list):
                 raise TypeError("Attempt to update key {} with a non list type".format(key))
             original[key] = update[key]
-        elif isinstance(original[key], int):
+        elif isinstance(original[key], (int, long)):
```

Comment:
Another way to do this is define something like:

```
integer_types = (int, long)
```

in one of the files and then use that everywhere you're doing integer testing.

As a side note, if you're using six, you can use `six.integer_types` which will do the right thing if you switch to Python 3.